app/timetables/tests/test_models.py

```diff
@@ -105,33 +106,70 @@ def test_duplicate_api_key_cannot_be_saved(self):
 
         self.assertRaises(ValidationError, self.another_timetable.save)
 
-    def test_current_cycle_day_greater_than_cycle_length_cannot_be_saved(self):
-        self.another_timetable.current_cycle_day = self.timetable.cycle_length + 1
+    def test_ref_cycle_day_greater_than_cycle_length_cannot_be_saved(self):
+        self.another_timetable.ref_cycle_day = self.timetable.cycle_length + 1
 
         self.assertRaises(ValidationError, self.another_timetable.save)
 
-    def test_cycle_length_and_current_cycle_day_of_zero_cant_be_saved(self):
+    def test_cycle_length_and_ref_cycle_day_of_zero_cant_be_saved(self):
         # test for cycle_length == 0
         self.another_timetable.cycle_length = 0
         self.assertRaises(ValidationError, self.another_timetable.save)
 
-        # test for current_cycle_day == 0
-        self.another_timetable.current_cycle_day = self.another_timetable.cycle_length
+        # test for ref_cycle_day == 0
+        self.another_timetable.ref_cycle_day = self.another_timetable.cycle_length
         self.another_timetable.cycle_length = self.timetable.cycle_length
 
         self.assertRaises(ValidationError, self.another_timetable.save)
 
-    def test_cycle_length_and_current_cycle_day_of_negative_value_cant_be_saved(self):
+    def test_cycle_length_and_ref_cycle_day_of_negative_value_cant_be_saved(self):
         # test for cycle_length < 0
         self.another_timetable.cycle_length = -3
         self.assertRaises(ValidationError, self.another_timetable.save)
 
-        # test for current_cycle_day < 0
-        self.another_timetable.current_cycle_day = self.another_timetable.cycle_length
+        # test for ref_cycle_day < 0
+        self.another_timetable.ref_cycle_day = self.another_timetable.cycle_length
         self.another_timetable.cycle_length = self.timetable.cycle_length
 
         self.assertRaises(ValidationError, self.another_timetable.save)
 
+    def test_calculate_cycle_day(self):
+        date = timezone.make_aware(timezone.datetime(2016, 11, 23, 12, 30, 0))
```

Comment:
@andela-kerinoso what if we change the variable name from `date` to something else? Just a thought.